### PR TITLE
Update gledopto.js

### PR DESCRIPTION
**Pull request by Shiftmaj**
Add support for GL-D-006P and GL-D-007P

### CODE REVIEW

File: devices/gledopto.js [hunk 1]
```diff
@@ -466,6 +466,22 @@ module.exports = [
...
+        description: 'Zigbee 6W Anti-glare Downlight RGB+CCT (pro)',
+        extend: gledoptoExtend.light_onoff_brightness_colortemp_color(),
+    },
+    {
+        zigbeeModel: ['GL-D-007P'],
+        model: 'GL-D-007P',
+        vendor: 'Gledopto',
+        ota: ota.zigbeeOTA,
+        description: 'Zigbee 12W Anti-glare Downlight RGB+CCT (pro)',
+        extend: gledoptoExtend.light_onoff_brightness_colortemp_color(),
```
Comment (Koenkk) [inline devices/gledopto.js:483]: Please add the color temperature range as described on https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html#_3-adding-converter-s-for-your-device

**Comment by Koenkk** [comment]
Note to self; after this merge

File: devices/gledopto.js (hunk 1 above)
Comment (Shiftmaj) [reply devices/gledopto.js]: I updated the file as requested, hope it's all good now 😃

File: devices/gledopto.js [hunk 2]
```diff
@@ -466,6 +466,20 @@ module.exports = [
         description: 'Zigbee 12W Downlight RGB+CCT (pro)',
         extend: gledoptoExtend.light_onoff_brightness_colortemp_color(),
     },
+    {
+        zigbeeModel: ['GL-D-006P'],
+        model: 'GL-D-006P',
+        vendor: 'Gledopto',
+        description: 'Zigbee 6W Anti-glare Downlight RGB+CCT (pro)',
+        extend: extend.light_onoff_brightness_colortemp_color({colorTempRange: [158, 495]}),
```
Comment (Koenkk) [inline devices/gledopto.js:474]: Did you use `extend` instead of `gledoptoExtend` on purpose?

Comment (Shiftmaj) [reply devices/gledopto.js:474]: Yes, not sure how I managed to get it working last week, but this week it definitely don't work. I have the following error while starting Z2M:
```Using '/app/data' as data directory
(node:7) UnhandledPromiseRejectionWarning: ReferenceError: gledoptoExtend is not defined
(Use `node --trace-warnings ...` to show where the warning was created)
(node:7) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:7) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.```

This is my first integration, so I may be doing something wrong...

For reference, here are my edited and working files:
/data/GLD006P.js
```const fz = require('zigbee-herdsman-converters/converters/fromZigbee');
const tz = require('zigbee-herdsman-converters/converters/toZigbee');
const exposes = require('zigbee-herdsman-converters/lib/exposes');
const reporting = require('zigbee-herdsman-converters/lib/reporting');
const extend = require('zigbee-herdsman-converters/lib/extend');
const e = exposes.presets;
const ea = exposes.access;

const definition = {
    zigbeeModel: ['GL-D-006P'],
    model: 'GL-D-006P',
    vendor: 'Gledopto',
    description: 'Zigbee 6W Anti-glare Downlight RGB+CCT (pro)',
    extend: extend.light_onoff_brightness_colortemp_color({colorTempRange: [158, 495]}),
};

module.exports = definition;```

/app/configuration.yaml
```homeassistant: false
permit_join: false
mqtt:
  base_topic: zigbee2mqtt
  server: mqtt://localhost:1884
  user: xxxx
  password: xxxx
serial:
  port: /dev/ttyACM0
frontend:
  port: 8080
map_options:
  graphviz:
    colors:
      fill:
        enddevice: '#fff8ce'
        coordinator: '#e04e5d'
        router: '#4ea3e0'
      font:
        coordinator: '#ffffff'
        router: '#ffffff'
        enddevice: '#000000'
      line:
        active: '#009900'
        inactive: '#994444'
devices:
  '0x84fd27fffebb92c4':
    friendly_name: '0x84fd27fffebb92c4'
  '0x84fd27fffe931816':
    friendly_name: '0x84fd27fffe931816'
advanced:
  log_level: debug
external_converters:
  - GLD006P.js```

Comment (Koenkk) [reply devices/gledopto.js]: This is because the `gledoptoExtend` is defined only in this file, I've changed it to `gledoptoExtend`, thanks!

**Comment by ajenjo85** [comment]
Add support for GL-S-006P. 
Thanks

**Pull request merged**